src/filetypes.c

```diff
@@ -951,7 +951,7 @@ static void load_settings(guint ft_id, GKeyFile *config, GKeyFile *configh)
 	{
 		ft->lang = tm_source_file_get_named_lang(result);
 		if (ft->lang < 0)
-			geany_debug("Cannot find tag parser '%s' for custom filetype '%s'.", result, ft->name);
+			geany_debug("Cannot find tags parser '%s' for custom filetype '%s'.", result, ft->name);
```

Comment:
By the way I'm definitely not thinking in terms of internal implementation - the whole patch is just to get rid of the internal implementation terms. However, the  fact is that we use ctags and we use ctags parsers which parse the sources and this is the point where the parsers get visible to the user because they need to be assigned to the given language and we can't hide that.